redis/src/main/scala/zio/redis/Output.scala

```diff
@@ -813,4 +813,35 @@ object Output {
         case other                          => throw ProtocolError(s"$other isn't an integer >= -1")
       }
   }
+
+  case object ClientInfoOutput extends Output[ClientInfo] {
+    protected def tryDecode(respValue: RespValue)(implicit _codec: Codec): ClientInfo = {
+      val decoder = ClientInfoCodec.decode(ClientInfo.schema)
+
+      respValue match {
+        case RespValue.BulkString(data) =>
+          decoder(data).fold(e => throw CodecError(e), identity)
+        case RespValue.SimpleString(data) =>
+          decoder(Chunk.fromArray(data.getBytes)).fold(e => throw CodecError(e), identity)
+        case other => throw ProtocolError(s"$other isn't a bulk string")
+      }
+    }
+  }
+
+  case object ClientListOutput extends Output[Chunk[ClientInfo]] {
+    protected def tryDecode(respValue: RespValue)(implicit _codec: Codec): Chunk[ClientInfo] =
```

Comment:
```suggestion
    protected def tryDecode(respValue: RespValue)(implicit codec: Codec): Chunk[ClientInfo] =
```

Reply:
updated this in the last commit (merged with master)